torch/fx/symbolic_trace.py

```diff
@@ -238,6 +238,60 @@ def proxy_placeholder(name: str):
 
         return root_fn, args
 
+    def call_wrapped_func(self, orig_fn, args, kwargs):
```

Comment:
re the discussion about this being a class method of tracer: if you want to override the behavior of how a wrapped function behaves, why isn't overriding `create_proxy` and doing something there equivalent? I feel like it would be better to handle all places were we would normally insert a call_function node the same rather than making wrapped functions behave differently than functions override by `__torch_function__`. In the future we may chose to figure out how to wrap execution differently (such as using python tracing to do it), and allowing different behavior based on how an implementation detail of a function is patched would be hard to adapt.

Reply:
@mrzzd